src/main/scala/org/broadinstitute/dsde/workbench/avram/util/ErrorResponse.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.workbench.avram.util
+
+import java.util.logging.Logger
+
+import com.google.api.server.spi.response._
+
+/**
+  * Representation of an error response for use throughout the code. Is eventually translated to one
+  * of the Google Cloud Endpoints Framework exceptions that, when thrown, results in the correct
+  * HTTP response.
+  *
+  * @param statusCode HTTP status code
+  * @param message    error message
+  */
+case class ErrorResponse(statusCode: Int, message: String) {
+  private val log = Logger.getLogger(getClass.getName)
+
+  /**
+    * Create a Google Cloud Endpoints Framework exception matching the status code.
+    */
+  def exception: Exception = {
+    log.info(message)
```

Comment:
`log.error`?

Reply:
In fact, I'm going to declare that this is not the place to log the message. This just creates an exception object. One would expect it wouldn't be created if it isn't going to be thrown but it's also not guaranteed. I'm going to leave it to the actual error handling code to log the error.